src/CarbonAware.WebApi/src/README.md

```diff
@@ -11,6 +11,8 @@ The Carbon Aware SDK provides an API to get the marginal carbon intensity for a
     - [GET emissions/bylocations/best](#get-emissionsbylocationsbest)
     - [GET emissions/forecasts/current](#get-forecastscurrent)
     - [POST emissions/forecasts/batch](#post-forecastsbatch)
+    - [GET emissions/average-carbon-intensity](#get-avgcarbonintensity)
```

Comment:
nit: Seems like the links are broken starting from `emissions/forecasts/current`. It would be nice to fix that.

Reply:
Fixed, just some small typos